resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/backend/minikube/Minikube.scala

```diff
@@ -126,17 +126,21 @@ private[spark] object Minikube extends Logging {
     }
   }
 
-  def executeMinikube(action: String, args: String*): Seq[String] = {
+  def describePods(labels: String): Seq[String] =
+    Minikube.executeMinikube(false, "kubectl", "--", "describe", "pods", "--all-namespaces",
+      "-l", labels)
```

Comment:
If you don't mind, could you move this function to the end? For example, after `minikubeServiceAction`?